tests/features/rhpam/businesscentral/rhpam-businesscentral.feature

```diff
@@ -9,7 +9,7 @@ Feature: RHPAM Business Central configuration tests
   Scenario: Check for product and version environment variables
     When container is ready
     Then run sh -c 'echo $JBOSS_PRODUCT' in container and check its output for rhpam-businesscentral
-     And run sh -c 'echo $RHPAM_BUSINESS_CENTRAL_VERSION' in container and check its output for 7.8.0
+     And run sh -c 'echo $RHPAM_BUSINESS_CENTRAL_VERSION' in container and check its output for 7.8
```

Comment:
Shouldn't be there checked micro version? So 7.8.1. If there is only minor version (7.8) in output, then it's OK.

Reply:
all other tests is only major version, but I do agree, will update others to test major+minor.

Reply:
updated.